Cargo.toml

```diff
@@ -5,13 +5,13 @@ authors = [
 ]
 name = "poppler"
 license = "GPL-2.0"
-version = "0.3.1"
+version = "0.4.0"
```

Comment:
As the enums (etc) where drastically changed, I figured it would be safer to increase the "major"ish ver.